src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        Random random = new Random();
+        int randomNumber = random.nextInt(100);
+        ball.setNumber(randomNumber);
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
1. Please make random and colorSupplier class variables
2. Please extract magic numbers to constants with proper naming